doc/glossary/index.rst

```diff
@@ -0,0 +1,33 @@
+.. _glossary:
+
+Glossary
+========
+
+.. glossary::
+   :sorted:
+
+   OTP Value
+        A one-time password, which is generated by some mathematical algorithm, usually HMAC,
+        based on a seed. The term OTP value is used frequently by privacyIDEA to distinguish
+        the changing value from the :term:`OTP PIN`.
+        In terms of two factor authentication the OTP Value is actually the 2nd factor,
+        the possion factor, since it is usually only possible to calculage, if the user
+        is in the possession of the smartphone app or a hardware token. The OTP Value is calculated
+        using the secret cryptographic :term:`Seed`.
+
+   OTP PIN
+        The OTP PIN is the secret password with which the user authenticates against privacyIDEA.
+        The policy action :ref:`otppin_policy` sets the type of password. With this
+        password privacyIDEA will identify the tokens for which further actions are taken
+        (trigger a challenge or check a given :term:`OTP Value`).
+        In terms of two factor authentication the OTP PIN is the first factor, the knowledge.
```

Comment:
We should add the information, that an OTP PIN is stored per token.
I.e. if a user has several tokens he can have different otp pins per token.
This is different if the otppinpolicy is set to userstore, when all tokens have the static factor from the user store